app/views/placements/schools/users/index.html.erb

```diff
@@ -1,13 +1,5 @@
-<% content_for(:page_title) { t(".users") } %>
-<% if current_user.organisation_count > 1 %>
-  <%= content_for(:header_content) do %>
-    <%= render(ContentHeaderComponent.new(
-      title: @organisation.name,
-      actions: [govuk_link_to(t(".change_organisation"), placements_organisations_path, no_visited_state: true)],
-    )) %>
-  <% end %>
-<% end %>
 <%= render "placements/primary_navigation", organisation: @organisation, current_navigation: :users %>
+<% content_for(:page_title) { t(".users") } %>
```

Comment:
```suggestion
<%= content_for(:page_title) { t(".users") } %>
```
Does this need to be like this?

Reply:
I don't think `=` is required here because it's not actively outputting something to the page. The return value of `content_for` is presumably `nil` because it performs a side-effect (i.e. setting a value for the layout) rather than actively causing output itself.

Reply:
In other words: I think they'll be functionally the same with or without the `=`.